tests/CMakeLists.txt

```diff
@@ -36,7 +36,21 @@ endif()
 # Compile options
 
 if (WIN32)
-    add_compile_options(-wd4100 -wd4514 -wd4668 -wd4710 -wd4711 -wd4820)
+    # 4100 - unreferenced formal parameter
+    # 4514 - unreferenced inline function has been removed
+    # 4625 - copy constructor was implicitly defined as deleted because a base class copy constructor is inaccessible or deleted
+    # 4626 - assignment operator was implicitly defined as deleted because a base class assignment operator is inaccessible or deleted
+    # 4668 - 'symbol' is not defined as a preprocessor macro, replacing with '0' for 'directives'
+    # 4710 - function not inlined
+    # 4711 - 'function' selected for inline expansion
+    # 4774 - format string expected in argument number is not a string literal
+    # 4820 - bytes padding added after construct 'member_name'
+    # 5025 - move assignment operator was implicitly defined as deleted
+    # 5026 - move constructor was implicitly defined as deleted
+    # 5027 - move assignment operator was implicitly defined as deleted
+    # 5039 - pointer or reference to potentially throwing function passed to extern C function under -EHc. Undefined behavior may occur if this function throws an exception.
+    add_compile_options(-wd4100 -wd4514 -wd4625 -wd4626 -wd4668 -wd4710 -wd4711 -wd4774 -wd4820 -wd5025 -wd5026 -wd5027 -wd5039)
```

Comment:
The comments are great.

But why do you need to disable more warnings (globally -- for all tests) instead of just fixing the problems in the code that lead to the warnings?

Reply:
@BruceForstall  That is a good question. In almost all of those cases, I just wasn't able to fix the issue - no matter what I tried. There were a few issues around exceptions that were in Windows headers that I could not mitigate at all. I could be more targeted for sure and just apply them to this project if that is preferred?